integration-tests/src/utils.rs

```diff
@@ -0,0 +1,222 @@
+        )))
+        .mount(&mock_server)
+        .await;
+
+    Mock::given(method("POST"))
+        .and(path("/graphql"))
+        .respond_with(ResponseTemplate::new(200).set_body_string(format!(
+            r#"{{
+                "data": {{
+                  "proofOfIndexing": "{poi}",
+                  "blockHashFromNumber":"4dbba1ba9fb18b0034965712598be1368edcf91ae2c551d59462aab578dab9c5",
+                  "indexingStatuses": [
+                    {{
+                      "subgraph": "{}",
+                      "synced": true,
+                      "health": "healthy",
+                      "fatalError": null,
+                      "chains": [
+                        {{
+                          "network": "mainnet",
+                          "latestBlock": {{
+                            "number": "{block_number}",
+                            "hash": "b30395958a317ccc06da46782f660ce674cbe6792e5573dc630978c506114a0a"
+                          }},
+                          "chainHeadBlock": {{
+                            "number": "{block_number}",
+                            "hash": "b30395958a317ccc06da46782f660ce674cbe6792e5573dc630978c506114a0a"
+                          }}
+                        }}
+                      ]
+                    }},
+                    {{
+                        "subgraph": "{}",
+                        "synced": true,
+                        "health": "healthy",
+                        "fatalError": null,
+                        "chains": [
+                          {{
+                            "network": "goerli",
+                            "latestBlock": {{
+                                "number": "{}",
+                                "hash": "b30395958a317ccc06da46782f660ce674cbe6792e5573dc630978c506114a0a"
+                              }},
+                              "chainHeadBlock": {{
+                                "number": "{}",
+                                "hash": "b30395958a317ccc06da46782f660ce674cbe6792e5573dc630978c506114a0a"
+                              }}
+                          }}
+                        ]
+                      }}
+                  ]
+                }}
+              }}
+              "#,
+              ipfs_hashes[0], ipfs_hashes[1], block_number + 5, block_number + 5, // use the provided ipfs hashes
+            )))
+        .mount(&mock_server)
+        .await;
+
+    mock_server.uri()
+}
+
+pub fn setup_mock_env_vars(mock_server_uri: &String) {
+    env::set_var(
+        "GRAPH_NODE_STATUS_ENDPOINT",
+        format!("{}{}", mock_server_uri, "/graphql"),
+    );
+
+    env::set_var(
+        "REGISTRY_SUBGRAPH_ENDPOINT",
+        format!("{}{}", mock_server_uri, "/graphcast-registry"),
+    );
+
+    env::set_var(
+        "NETWORK_SUBGRAPH_ENDPOINT",
+        format!("{}{}", mock_server_uri, "/network-subgraph"),
+    );
+}
+
+pub struct RadioRuntimeConfig {
```

Comment:
Do you think these could maybe be renamed? The folder structure is clear that these config is for the tests, but I'm worried as it sounds like config for normal runtime. Maybe something like...`RadioTestConfig`?

Reply:
agreed

Reply:
also docstrings would be helpful I suppose

Reply:
✅